src/tests/simulation/src/cluster.rs

```diff
@@ -259,6 +259,8 @@ metrics_level = "Disabled"
             meta_nodes: 1,
             compactor_nodes: 1,
             compute_node_cores: 1,
+            per_session_queries: vec!["SET STREAMING_ENABLE_ARRANGEMENT_BACKFILL = true;".into()]
```

Comment:
This should not have any impact on other tests, right?

Reply:
Yes, it won't affect other tests.